tests/test_device_initialization.py

```diff
@@ -24,357 +24,565 @@
 from qiskit.providers.aer.noise import NoiseModel
 from qiskit.providers.models import BackendProperties
 
-from defaults import pennylane as qml, BaseTest
+from defaults import pennylane as qml
+from defaults import ARGS as args
 from pennylane_qiskit import IbmQQiskitDevice, AerQiskitDevice, BasicAerQiskitDevice
 
-log.getLogger('defaults')
-
-
-class DeviceInitialization(BaseTest):
-    """test aspects of the device initialization.
-    """
-
-    num_subsystems = 4
-    devices = None
-
-    def test_ibm_no_token(self):
-        # if there is an IBMQX token, save it and unset it so that it doesn't interfere with this test
-        token_from_environment = os.getenv('IBMQX_TOKEN')
-        if token_from_environment is not None:
-            del os.environ['IBMQX_TOKEN']
-
-        if self.args.device == 'ibmq' or self.args.device == 'all':
-            self.assertRaises(ValueError, IbmQQiskitDevice, wires=self.num_subsystems,
-                              msg='Expected a ValueError if no IBMQX token is present.')
+import pytest
+
+
+num_subsystems = 4
+devices = None
+
+
+def test_ibm_no_token():
+    # if there is an IBMQX token, save it and unset it so that it doesn't interfere with this test
+    token_from_environment = os.getenv("IBMQX_TOKEN")
+    if token_from_environment is not None:
+        del os.environ["IBMQX_TOKEN"]
```

Comment:
How come you're deleting the IBMQX token environment variable here?

Reply:
Oh, I see, it's so that the next text will raise the correct error!

Reply:
Yeah. Also, Carsten wrote those tests already.